.github/workflows/ci.yaml

```diff
@@ -8,21 +8,44 @@ on:
     branches:
       - master
 
+defaults:
+  run:
+    shell: 'bash -o errexit -o nounset -o pipefail {0}'
```

Comment:
Why would you do that?

Reply:
Because what's often call the "strict" mode of Bash isn't on by default.

Reply:
Yeah but the bash is terminated right afterwards. It does not affect subsequent runs.

Reply:
For `errexit`, It's true it won't affect other runs or jobs, but it would affect subsequent commands in the same job.
At the moment, we only have a single command, so it doesn't add any value, but if there were ever a second command added, the second command would execute even if the first command failed. That's usually not desirable.

`nounset` was there to prevent us from accidentally using a variable that wasn't set.

`pipefail` is another one that doesn't matter right now because we've only got 1 command taking from a pipe (and it's one I added with this PR).

I just had added this since it's what I put on all my Bash scripts (unless I have a specific reason not to).